### PR TITLE
2.13.4/2.14.0 draft release notes

### PR DESCRIPTION
**Pull request by joshmoore**
Kicking off the release process. Comments on which scope ("maintenance", "major", etc) each PR should belong to welcome. Thanks to @MSanKeys963 for drafting this over the holidays! :tada:

### CODE REVIEW

**Comment by joshmoore** [comment]
@MSanKeys963: thoughts on adding a blurb at the top of the release with a pointer to one or more of the outreachy blogs? Also, we'll need to think about the contributors list.

**Comment by MSanKeys963** [comment]
Thanks for sending the PR, @joshmoore.

> thoughts on adding a blurb at the top of the release with a pointer to one or more of the outreachy blogs?

Are you mentioning this one: https://zarr.dev/blog/welcoming-outreachy-2022-interns/?
Or @caviere's and @DON-BRAN's blogs?

> Also, we'll need to think about the contributors list.

There's some work done by @GbotemiB in #1185 for this. @GbotemiB do you wish to complete this PR anytime soon?

File: docs/release.rst [hunk 1]
```diff
@@ -6,21 +6,57 @@ Release notes
...
+
+Major changes
+~~~~~~~~~~~~~
+
+`NONE`
+
+Enhancements
+~~~~~~~~~~~~
+
+* Handle fsspec.FSMap using FSStore store
```
Comment (MSanKeys963) [inline docs/release.rst:30]:
```suggestion
* Handle fsspec.FSMap using FSStore store.
```

File: docs/release.rst [hunk 2]
```diff
@@ -6,21 +6,57 @@ Release notes
...
+Documentation
+~~~~~~~~~~~~~
+
+* Fix minor indexing errors in tutorial and specification examples of documentation. 
+  By :user:`Kola Babalola <sprynt001>` :issue:`1277`.
+
+* Add `requirements_rtfd.txt` in `contributing.rst`.
+  By :user:`AWA BRANDON AWA <DON-BRAN>` :issue:`1243`.
+
+* Add documentation for find/findall using visit
```
Comment (MSanKeys963) [inline docs/release.rst:55]:
```suggestion
* Add documentation for find/findall using visit.
```

File: docs/release.rst [hunk 3]
```diff
@@ -6,21 +6,57 @@ Release notes
...
+  By :user:`Weddy Gikunda <caviere>` :issue:`1241`.
+
+* Refresh of the main landing page
```
Comment (MSanKeys963) [inline docs/release.rst:58]:
```suggestion
* Refresh of the main landing page.
```

File: docs/release.rst [hunk 4]
```diff
@@ -55,6 +91,74 @@ Maintenance
 * Delete unused files.
   By :user:`John Kirkham <jakirkham>` :issue:`1251`.
 
+* Skip labeler for bot PRs
```
Comment (MSanKeys963) [inline docs/release.rst:94]:
```suggestion
* Skip labeller for bot PRs.
```

File: docs/release.rst [hunk 5]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`Saransh Chopra <Saransh-cpp>` :issue:`1271`.
+
+* Restore Flake8 configuration
```
Comment (MSanKeys963) [inline docs/release.rst:97]:
```suggestion
* Restore Flake8 configuration.
```

File: docs/release.rst [hunk 6]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`John Kirkham <jakirkham>` :issue:`1249`.
+
+* Add missing newline at EOF
```
Comment (MSanKeys963) [inline docs/release.rst:100]:
```suggestion
* Add missing newline at EOF.
```

File: docs/release.rst [hunk 7]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`Dimitri Papadopoulos` :issue:`1253`.
+
+* Add `license_files` to `pyproject.toml`
```
Comment (MSanKeys963) [inline docs/release.rst:103]:
```suggestion
* Add `license_files` to `pyproject.toml`.
```

File: docs/release.rst [hunk 8]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`John Kirkham <jakirkham>` :issue:`1247`.
+
+* Adding `pyupgrade` suggestions
```
Comment (MSanKeys963) [inline docs/release.rst:106]:
```suggestion
* Adding `pyupgrade` suggestions.
```

File: docs/release.rst [hunk 9]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+* Fixed some linting errors.
+  By :user:`Weddy Gikunda <caviere>` :issue:`1226`.
+
+* Added the link to main website in readthedocs sidebar.
+  By :user:`Stephanie_nkwatoh <steph237>` :issue:`1216`.
+
+* Remove redundant wheel dependency in `pyproject.toml`
+  By :user:`Dimitri Papadopoulos Orfanos <DimitriPapadopoulos>` :issue:`1233`.
+
+* Turned on `isloated_build` in `tox.ini` file
```
Comment (MSanKeys963) [inline docs/release.rst:118]:
```suggestion
* Turned on `isloated_build` in `tox.ini` file.
```

File: docs/release.rst [hunk 10]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`AWA BRANDON AWA <DON-BRAN>` :issue:`1210`.
+
+* Fixed `flake8` alert and avoid duplication of `Zarr Developers`.
+  By :user:`Dimitri Papadopoulos Orfanos <DimitriPapadopoulos>` :issue:`1203`.
+
+* Bump to NumPy 1.20+ in `environment.yml`
```
Comment (MSanKeys963) [inline docs/release.rst:124]:
```suggestion
* Bump to NumPy 1.20+ in `environment.yml`.
```

File: docs/release.rst [hunk 11]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`John Kirkham <jakirkham>` :issue:`1201`.
+
+* Bump to NumPy 1.20 in `pyproject.toml`
```
Comment (MSanKeys963) [inline docs/release.rst:127]:
```suggestion
* Bump to NumPy 1.20 in `pyproject.toml`.
```

File: docs/release.rst [hunk 12]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`Dimitri Papadopoulos Orfanos <DimitriPapadopoulos>` :issue:`1192`.
+
+* Remove LGTM (`.lgtm.yml`) configuration file
```
Comment (MSanKeys963) [inline docs/release.rst:130]:
```suggestion
* Remove LGTM (`.lgtm.yml`) configuration file.
```

File: docs/release.rst [hunk 13]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`Dimitri Papadopoulos Orfanos <DimitriPapadopoulos>` :issue:`1191`.
+
+* Codespell will skip `fixture` in pre-commit
```
Comment (MSanKeys963) [inline docs/release.rst:133]:
```suggestion
* Codespell will skip `fixture` in pre-commit.
```

File: docs/release.rst [hunk 14]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`Dimitri Papadopoulos Orfanos <DimitriPapadopoulos>` :issue:`1197`.
+
+* Add msgpack in `requirements_rtfd.txt`
```
Comment (MSanKeys963) [inline docs/release.rst:136]:
```suggestion
* Add msgpack in `requirements_rtfd.txt`.
```

File: docs/release.rst [hunk 15]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`Emmanuel Bolarinwa <GbotemiB>` :issue:`1188`.
+
+* Added license to docs fixed a typo from `_spec_v2` to `_spec_v3`.
+  By :user:`AWA BRANDON AWA <DON-BRAN>` :issue:`1182`.
+
+* Fixed installation link in `README.md`
```
Comment (MSanKeys963) [inline docs/release.rst:142]:
```suggestion
* Fixed installation link in `README.md`.
```

File: docs/release.rst [hunk 16]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`AWA BRANDON AWA <DON-BRAN>` :issue:`1177`.
+
+* Fixed typos in `installation.rst` and `release.rst`
```
Comment (MSanKeys963) [inline docs/release.rst:145]:
```suggestion
* Fixed typos in `installation.rst` and `release.rst`.
```

File: docs/release.rst [hunk 17]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`Chizoba Nweke <zobbs-git>` :issue:`1178`.
+
+* Set `docs/conf.py` language to `en`
```
Comment (MSanKeys963) [inline docs/release.rst:148]:
```suggestion
* Set `docs/conf.py` language to `en`.
```

File: docs/release.rst [hunk 18]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`AWA BRANDON AWA <DON-BRAN>` :issue:`1174`.
+
+* Added `installation.rst` to the docs
```
Comment (MSanKeys963) [inline docs/release.rst:151]:
```suggestion
* Added `installation.rst` to the docs.
```

File: docs/release.rst [hunk 19]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`AWA BRANDON AWA <DON-BRAN>` :issue:`1170`.
+
+* Adjustment of year to `2015-2018` to `2015-2022` in the docs
```
Comment (MSanKeys963) [inline docs/release.rst:154]:
```suggestion
* Adjustment of year to `2015-2018` to `2015-2022` in the docs.
```

File: docs/release.rst [hunk 20]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`Emmanuel Bolarinwa <GbotemiB>` :issue:`1165`.
+
+* Updated `Forking the repository` section in `contributing.rst`
```
Comment (MSanKeys963) [inline docs/release.rst:157]:
```suggestion
* Updated `Forking the repository` section in `contributing.rst`.
```

File: docs/release.rst [hunk 21]
```diff
@@ -55,6 +91,74 @@ Maintenance
...
+  By :user:`AWA BRANDON AWA <DON-BRAN>` :issue:`1171`.
+
+* Updated GitHub actions
```
Comment (MSanKeys963) [inline docs/release.rst:160]:
```suggestion
* Updated GitHub actions.
```

**Comment by MSanKeys963** [comment]
Suggested a few minor edits above.

**Comment by joshmoore** [comment]
> Are you mentioning this one: https://zarr.dev/blog/welcoming-outreachy-2022-interns/?

Yes, since there were lots of contributions from the wider outreachy community.

File: docs/release.rst [hunk 22]
```diff
@@ -6,20 +6,57 @@ Release notes
...
 
+.. _release_2.13.4:
+
+2.13.4
+------
+
+Major changes
+~~~~~~~~~~~~~
+
+`NONE`
```
Comment (MSanKeys963) [inline docs/release.rst:25]:
```suggestion
Appreciation
~~~~~~~~~~~~~

Special thanks to Outreachy participants for contributing to most of the maintenance PRs. Please read the blog post summarising the contribution phase and welcoming new Outreachy interns: https://zarr.dev/blog/welcoming-outreachy-2022-interns/

```

Comment (MSanKeys963) [reply docs/release.rst:25]: Please have a look, @joshmoore. ^

**Comment by MSanKeys963** [comment]
@jakirkham, do you think we can get #1285 in this release?

**Comment by joshmoore** [comment]
I'm going to review the rendered outreachy blurb, @MSanKeys963, and then go ahead and move forward with this as a patch release. `ensure_bytes` can follow up either as another quick patch release or as a minor release.

**Comment by rabernat** [comment]
Why not get #1285 in? This is really important to our contributors at scalable minds.

**Comment by joshmoore** [comment]
It's on @jakirkham's list to review, and the release notes were done for the things merged so it makes sense to get those out before things get tricky again. (Releases are easy; notes are [currently] hard)

**Pull request merged**